doc/syslog/syslog-design.md

```diff
@@ -291,26 +362,84 @@ yes - set source IP, no - generate error
 **Example:**
 ```
 Default VRF:
-*.* action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1")
+*.notice action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1")
 MGMT VRF:
-*.* action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1" device="mgmt")
+*.notice action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1" device="mgmt")
 DATA VRF:
-*.* action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1" device="Vrf-Data")
+*.notice action(type="omfwd" target="2.2.2.2" protocol="udp" address="1.1.1.1" device="Vrf-Data")
+```
+
+#### 3.3.4.5. Filter/Trap: `set/set`
+
+Log regex filter
+
+Filter:
```

Comment:
Is it possible to configure both include & exclude regex simultaneously? Please capture the restrictions in the design.

Reply:
not possible, added restriction in the document.